packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -178,10 +174,6 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
     if (this.tor && !options.saveTor) {
       await this.tor.kill()
     }
-    if (this.registrationService) {
-      this.logger('Stopping registration service')
-      await this.registrationService.stop()
-    }
```

Comment:
Because in the old model this service needed to be initialized - start listening on express server, add routing etc. Now it has just methods, so 'stop' method became empty.